src/configuration/api/api.configuration.ts

```diff
@@ -0,0 +1,5 @@
+export const FILE_API_UPLOAD = "/api/v3/file/upload/";
```

Comment:
const API = {
    FILE_UPLOAD = "/api/v3/file/upload/",
    LOGIN_REDIRECT = "/login?redirect=/tldraw?roomName="
    ...
}

API.FILE_UPLOAD 

Also only a questions what do you like more